kratos/identity.schema.json

```diff
@@ -24,10 +24,21 @@
           "pattern": "^[\\w\\-]+$",
           "maxLength": 32
         },
-        "description": { "type": "string" },
-        "motivation": { "type": "string" },
-        "profile_image": { "type": "string" },
-        "language": { "type": "string" },
+        "optedNewsletterIn": {
```

Comment:
Could maybe be named newsletterSubscription but it is understandable either way.